reference/halyard/commands.md

```diff
@@ -4509,9 +4509,13 @@ hal config security authn ldap edit [parameters]
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--no-validate`: (*Default*: `false`) Skip validation.
  * `--url`: ldap:// or ldaps:// url of the LDAP server
- * `--user-dn-pattern`: Placeholder...uid={0},ou=users
- * `--user-search-base`: Placeholder...
- * `--user-search-filter`: Placeholder
+ * `--user-dn-pattern`: The pattern for finding a user's DN using simple pattern matching. For
```

Comment:
This page is auto-generated, I think you want to make these changes in halyard?

Reply:
The changes should be made here: https://github.com/spinnaker/halyard/blob/master/halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/ldap/EditLdapCommand.java#L52-L64

Run `make` in the `halyard-cli` directory to regenerate the local docs. The next stable release of halyard will automatically update this page.